internal/patroni/reconcile.go

```diff
@@ -150,10 +148,8 @@ func instanceProbes(cluster *v1beta1.PostgresCluster, container *corev1.Containe
 	// TODO(cbandy): Consider if a PreStop hook is necessary.
 	container.LivenessProbe = probeTiming(cluster.Spec.Patroni)
 	container.LivenessProbe.InitialDelaySeconds = 3
-	container.LivenessProbe.HTTPGet = &corev1.HTTPGetAction{
-		Path:   "/liveness",
-		Port:   intstr.FromInt(int(*cluster.Spec.Patroni.Port)),
-		Scheme: corev1.URISchemeHTTPS,
+	container.LivenessProbe.Exec = &corev1.ExecAction{
+		Command: []string{"/usr/local/bin/postgres-liveness-check.sh"},
```

Comment:
Maybe we want to do this change only for >2.7.0 🤔

Reply:
done here: https://github.com/percona/percona-postgresql-operator/pull/1099/commits/c377050ea78d60dd77772eb3de23eeec90f441f7